docs/fsharp/style-guide/conventions.md

```diff
@@ -641,3 +641,19 @@ module Networking =
 ```
 
```

Comment:
line 640 - unrelated to your changes - my OCD is telling me those periods should be aligned 😉